src/js/components/DataTable/Body.js

```diff
@@ -181,20 +182,26 @@ const Body = forwardRef(
 
     return (
       <Keyboard
-        onEnter={
-          // active is undefined if user never used the keyboard
-          onClickRow &&
-          active >= 0 &&
-          (!disabled ||
-            !disabled.includes(datumValue(data[active], primaryProperty)))
-            ? (event) => {
-                event.persist();
-                const adjustedEvent = event;
-                adjustedEvent.datum = data[active];
-                onClickRow(adjustedEvent);
-              }
-            : undefined
-        }
+        onEnter={(event) => {
```

Comment:
The WCAG recommendation for checkboxes is to select them with "Space" (https://www.w3.org/TR/wai-aria-practices-1.1/examples/checkbox/checkbox-1/checkbox-1.html). That is how datatable keyboard selection currently works. So this would be a breaking change to only support "Enter" for selection.

Proposing that we support "Space" also for the selection when `onClickRow="onSelect"`.

Reply:
I'm glad you brought this up I was thinking the same especially since the normal behavior of `checkbox` is a "Space" bar I will add this with a comment